CHANGELOG.md

```diff
@@ -1,20 +1,53 @@
 # nf-core/hic: Changelog
 
-## v1.1.1
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## v1.2.0dev - 2020-05-12
+
+### `Added`
+
+* Bump v1.2.0dev
+* Merge template nf-core 1.9
+* Move some options to camel_case
+* Update conda environment file
```

Comment:
It's nice to explicitly list _which_ packages changed in the changelog. See the [methylseq release](https://github.com/nf-core/methylseq/blame/master/CHANGELOG.md#L14-L23) for an example.